### ISSUE TITLE
add extension trust in create backup

### ISSUE THREAD

**Issue by Character95** [mihonapp/mihon#240]
### Describe your suggested feature

when you need to uninstall install or update extension you can just restore extension trust through backup

### Other details

-1h enabling extension trust

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.16.1](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by AntsyLich** [comment]
Unneeded after #320
